packages/tuxguitar/PKGBUILD

```diff
@@ -0,0 +1,118 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Stephan Springer <buzo+arch@Lini.de>
+# Contributor: Thanos Apostolou <thanosapostolou@outlook.com>
+# Contributor: Alexandre Moine <alexandre@moine.me>
+# Contributor: Sergej Pupykin <pupykin.s+arch@gmail.com>
+# Contributor: |AhIoRoS| < ahioros@gmail.com >
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+pkgbase=tuxguitar
+pkgname=tuxguitar
```

Comment:
When this is changed to not be a split package anymore, `conflicts=(tuxguitar-common and tuxguitar-gtk)` must be added, otherwise you get errors like these on install, if you had the AUR package installed before:

    tuxguitar: /usr/share/applications/tuxguitar.desktop exists in filesystem (owned by tuxguitar-common)